### ISSUE TITLE
screen reader for support dart pad

### ISSUE THREAD

**Issue by adil-s** [dart-lang/dart-pad#1072]
Hi there!
I've just tried  out dart pad and it is not accessible for screen reader user. 
when I press enter on multi line edit box, the screen reader says"blank". I'm  sure that there is code in edit box because if I try selecting and copying of code and then pasting the code in to empty text file, I'm able to paste it. 
the most accessible web editor for screen reader is Monaco Editor.
I would be truly grateful if you could embed the Monaco Editor  in to dart pad or let me switch between default dart pad and Monaco editor based dart pad.
Monaco editor link:
https://microsoft.github.io/monaco-editor/index.html
OS: windows 10
browser: chrome
screen reader: NVDA

steps to reproduce the issue:
1. turn on the screen reader and open the dart pad 
2. type something in edit box and try reading it using arrow keys .

**Comment by RedBrogdon** [comment]
Thanks for filing an issue. I just created a label specifically for accessibility-related stuff, since this topic will only grow in importance as DartPad continues to evolve.

A quick scan of the open issues for CodeMirror leads me to believe that accessibility is not a solved issue for them:

https://github.com/codemirror/CodeMirror/issues/4604

On the other hand, I know that integrating Monaco would not be a simple thing to do and that when the subject has come up before there were some specific roadblocks mentioned. @domesticmouse, can you recall what some of them were?
